LLMonFHIR/FHIR Views/FHIRResourcesView.swift

```diff
@@ -40,41 +40,42 @@ import SwiftUI
 struct FHIRResourcesView<ContentView: View, ActionView: View>: View {
     @Environment(FHIRStore.self) private var fhirStore
     @State private var searchText = ""
-    
+    @State private var showAllItems: [String: Bool] = [:]
+
     private let navigationTitle: Text
     private let contentView: ContentView
     private let actionView: ActionView
-    
-    
+
+
     var body: some View {
         List {
             Section {
                 contentView
             }
-            
+
             if searchText.isEmpty {
                 Section {
                     actionView
                 }
             }
-            
+
             if fhirStore.allResources.filterByDisplayName(with: searchText).isEmpty {
                 Text("FHIR_RESOURCES_EMPTY_SEARCH_MESSAGE")
             } else {
                 resourcesSection
             }
-            
+
             Section { } footer: {
                 Text("Total Number of Resources: \(fhirStore.allResources.count)")
             }
         }
-            .searchable(text: $searchText)
-            .navigationDestination(for: FHIRResource.self) { resource in
-                InspectResourceView(resource: resource)
-            }
-            .navigationTitle(navigationTitle)
+        .searchable(text: $searchText)
```

Comment:
Please keep the ident here to match the overall Spezi code style :)

Reply:
Is it intentional that the "." is not aligned with "}"? 👀